### DISCUSSION TITLE
How long to get my first post unblocked by admins?

### DISCUSSION THREAD

**Discussion by lzap** [community/community#24034] [Other Feature Feedback, Questions, & Ideas]
<p>Hello,</p>
<p>I have sent my first post, it contain an example and link to our GH page and Akismet blocked it. I can no longer see it which is a shame because it took me a while to write it. This was 4 days ago.</p>
<p>Can admins take a look on this please? Thanks!</p>
<p><a href="https://github.community/t/sharing-a-variable-across-jobs/210565/1">https://github.community/t/sharing-a-variable-across-jobs/210565/1</a></p>

**Comment by nethgato** [comment, marked as answer]
<p>Hey <a class="mention" href="/lzap">@lzap</a> – apologies for the delay there. Your post got caught by the automated mod as potential spam.</p>
<p>I removed the flag and your post should now be active.</p>
